scholia/app/templates/topic_missing.html

```diff
@@ -69,5 +143,16 @@ <h2>Missing author items</h2>
 
 <table class="table table-hover" id="missing-author-items"></table>
 
+<h2>Missing publication date</h2>
+
+For the listed publications, Wikidata does not have publication dates.
+<table class="table table-hover" id="missing-pub-date"></table>
+
+
+<h2>Missing publication venue</h2>
+
+For the listed publications, Wikidata does not have pa statement as to where they were published.
```

Comment:
Is there a typo in "does not have pa statement"?

Reply:
Yes. Fixed.